buildSrc/src/main/java/com/google/android/material/composethemeadapter/dependencies.kt

```diff
@@ -25,10 +25,10 @@ object Libs {
 
     const val gradleMavenPublishPlugin = "com.vanniktech:gradle-maven-publish-plugin:0.12.0"
 
-    const val mdc = "com.google.android.material:material:1.1.0"
+    const val mdc = "com.google.android.material:material:1.2.1"
```

Comment:
Ditto, this could be a breaking change for apps right? (since 1.2.0 contained breaking changes). I vote to keep the library version dependency as low as possible.